src/source-pcap-file-helper.c

```diff
@@ -80,7 +80,7 @@ void PcapFileCallbackLoop(char *user, struct pcap_pkthdr *h, u_char *pkt)
     p->ts = SCTIME_FROM_TIMEVAL_UNTRUSTED(&h->ts);
     SCLogDebug("p->ts.tv_sec %" PRIuMAX "", (uintmax_t)SCTIME_SECS(p->ts));
     p->datalink = ptv->datalink;
-    p->pcap_cnt = ++pcap_g.cnt;
+    PcapPacketCntSet(p, ++pcap_g.cnt);
```

Comment:
here we don't need to use the wrapper, this code can just access the union. It then also bypasses the runmode checks which are redundant here